### ISSUE TITLE
Reboot after RSSI menu.

### ISSUE THREAD

**Issue by Jumbotroll** [talentraspel/GD-77#18]
GD-77 Reboot after Menu -> RSSI -> Red Button -> Arrow Up twice. 
Check video.

http://www.jadahl.com/GD-77_Reboot.mp4

Keep up the great work!

73 LB9AB
Jørn

**Comment by rogerclarkmelbourne** [comment]
It looks like somehow this is being caused by the reuse of the menu global variables, because gMenusEndIndex seemed to somehow get set to 62 even when there weren't that menu items in the menu.

These vars were re-initialised when the menuDisplayMenuList gets called to initalise, after the RSSI screen is closed, but somehow the  gMenusEndIndex was getting set to another value

This may be something to do with the RTOS, but I can't see how its happening, because I thought the RSSI screen and the menuDusplayMenuList function are both in the same task.

Anyway. I've pushed a fix for this in my fork, and I will make a Pull Request when I've fixed some of the other bugs
